test_vm/vendor/cookbooks/dbfit_test/recipes/diskspace.rb

```diff
@@ -0,0 +1,19 @@
+chef_gem 'di-ruby-lvm'
+
+lvm_physical_volume '/dev/sdb' do
+  action :create
+end
+
+lvm_volume_group 'centos' do
+  action :extend
+  physical_volumes ['/dev/sda2', '/dev/sdb']
```

Comment:
`/dev/sda2' is the existing LVM physical volume that comes with the CentOS 7.1. box.

I'm not sure if I need to specify all of the volumes, including existing volumes, in the volume group or to only specify the new ones. I'll try removing `/dev/sda2'.